### PR TITLE
Add contents from cookiecutter template

### PR DESCRIPTION
**Pull request by weiglszonja**

### CODE REVIEW

File: .github/workflows/test-install.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - name: Install pip
+      run: |
+        python -m pip install --upgrade pip
+        pip3 install packaging
+    - name: Install package
+      run: pip install -e .
+    - name: Test module load
+      uses: jannekem/run-python-script-action@v1
+      with:
+        script: import {{cookiecutter.repo_name_slug}}
```
Comment (CodyCBakerPhD) [inline .github/workflows/test-install.yml:30]: Looks like this one didn't make it in cookiecutter

File: src/pinto_lab_to_nwb/into_the_void/into_the_void_requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+nwb-conversion-tools==0.11.1   # Example of specific pinned dependecy
+some-extra-package==1.11.3  # Example of another extra package that's necessary for the current conversion
```
Comment (CodyCBakerPhD) [inline src/pinto_lab_to_nwb/into_the_void/into_the_void_requirements.txt:2]: Can you remove these?

File: .github/workflows/test-install.yml (hunk 1 above)
Comment (weiglszonja) [reply .github/workflows/test-install.yml:30]: Weird, I didn't make any modifications just run cookiecutter following the instructions.

Comment (CodyCBakerPhD) [reply .github/workflows/test-install.yml:30]: Can you fix it? Then we can merge this

**Pull request merged**